app/services/claims/claim/remove_empty_mentor_training_hours.rb

```diff
@@ -0,0 +1,15 @@
+class Claims::Claim::RemoveEmptyMentorTrainingHours
+  include ServicePattern
+
+  def initialize(claim:)
+    @claim = claim
+  end
+
+  def call
+    claim.mentor_trainings.without_hours.destroy_all
+  end
+
+  private
+
+  attr_reader :claim
+end
```

Comment:
As a relative outsider without context on this side of the codebase, it seems a bit odd to me that MentorTraining records are being created without hours if they're deemed to be 'bad data' that subsequently needs cleaning up. I would have thought a more sensible approach would be to validate the data and avoid creating 'invalid' records in the first place.

Reply:
I think this is a side effect of a multi-step form. We create a mentor training record when the user adds a mentor to a claim and then he needs to add hours to that mentor. There is an edge case where he can click the back link and back off from adding hours of training, so the mentor training needs to be removed